app/gateway/changelog.md

```diff
@@ -896,6 +896,16 @@ openid-connect
 * Bumped `lodash` for Dev Portal from 4.17.11 to 4.17.21
 * Bumped `lodash` for Kong Manager from 4.17.15 to 4.17.21
 
+## 2.8.2.2
+**Release Date** 2022/12/01
+
+### Fixes 
+
+#### Core
+
+* Fixed the timer error `lua_max_running_timers are not enough`.
```

Comment:
Seeing a `lua_max_running_timers are not enough` error is almost always a symptom and not a cause (and it can have _many_ causes). Is there a source for this with more detail?

Reply:
I don't think that change was supposed to be in this PR. It should only contain the last commit (https://github.com/Kong/docs.konghq.com/pull/4849/commits/c59e2a26b0e6850b989786158a60f84c7a050cb4). But that is a good question.